eclipse-theia/theia-blueprint
# Change license and copyright headers to MIT

#### What it does

Replaces the EPL license and notice files, as well as the package.json metadata and the copyright headers, to MIT.
This has been declared and approved with the community and the Eclipse Foundation (see links below).

Fixes https://github.com/eclipse-theia/theia-blueprint/issues/310
See also https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3875

#### How to test
  * Make sure all occurrences of EPL have been replaced accordingly
  * Make sure the linter check for the license works now for MIT headers
  * Make sure there is no functional change in the source code due to the license replacement

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
